sqrl-math/src/main/java/com/datasqrl/math/distribution/binomial_distribution.java

```diff
@@ -0,0 +1,14 @@
+package com.datasqrl.math.distribution;
+
+import com.google.auto.service.AutoService;
+import org.apache.flink.table.functions.ScalarFunction;
+
+/** Calculates the probability mass function for a binomial distribution. */
+@AutoService(ScalarFunction.class)
+public class binomial_distribution extends ScalarFunction {
```

Comment:
Minor: I think you can keep the classname capitalized, i.e. `Binominal_distribution` since SQL is case insensitive. Then it looks more "java".

Reply:
If you don’t feel strongly, I’d leave it as is. It’s better to keep it one way or the other, but not a mix, as that can be confusing.

Reply:
If you don’t feel strongly, I’d leave it as is. It’s better to keep it one way or the other, but not a mix, as that can be confusing. @mbroecheler 
I'm merging it for now, but feel free to ping me if you don't agree.